packages/shared/topology/utils/AlertFilters.ts

```diff
@@ -0,0 +1,32 @@
+import { Alert, K8sModel } from '@openshift-console/dynamic-plugin-sdk';
+import * as _ from 'lodash-es';
+import { NodeModel } from '../../models';
+
+export const commonFilter =
+  (name: string) =>
+  (alert: Alert): boolean => {
+    return (
+      _.get(alert, 'annotations.description', '').includes(name) ||
+      _.get(alert, 'annotations.message', '').includes(name)
+    );
+  };
+export const nodeFilter = (name: string) => (alert: Alert) =>
+  alert?.labels?.node === name || commonFilter(name)(alert);
+
+export const filterRelevantAlerts = (
+  resourceName: string,
+  resourceModel: K8sModel,
+  alerts: Alert[]
+): Alert[] => {
+  if (!resourceName || !!alerts) return [];
```

Comment:
Shouldn't we check is there are no alerts?
```suggestion
  if (!resourceName || _.isEmpty(alerts)) return [];
```

Reply:
ack